docs/how-to-write-contract.md

```diff
@@ -79,69 +92,55 @@ Thus, `Ledger` might throw some runtime (unchecked) exceptions in case it can no
 
 ### Determinism
 
-One very important thing to note when you write a contract for Scalar DL is that you have to make the contract deterministic.
-In other words, a contract must always produce the same output for a given particular input.
-The reason why determinism is important is that Scalar DL utilizes this property to detect tampering.
-For example, Scalar DL will lazily traverse assets and re-execute contracts to check if there is no discrepancy between the expected outcome and the actual data stored in the ledger.
-It also utilizes determinism to make the states of multiple independent Scalar DL components, which are possibly managed by different organizations, the same.
+One very important thing to note when you write a contract for Scalar DL is that you have to make the contract deterministic. In other words, a contract must always produce the same output for a given particular input. The reason why determinism is important is that Scalar DL utilizes this property to detect tampering.
+
+For example, Scalar DL will lazily traverse assets and re-execute contracts to check if there is no discrepancy between the expected outcome and the actual data stored in the ledger. It also utilizes determinism to make the states of multiple independent Scalar DL components (i.e., Ledger and Auditor), which are preferably managed by different organizations, the same.
```

Comment:
Very minor, but I like to remove the "which are preferably ..." part like the following for the readability since "the same" is a little far from "the states" and the administrative domain is off-topic here.

It also utilizes determinism to make the states of multiple independent Scalar DL components (i.e., Ledger and Auditor) the same.

Reply:
Thank you! Updated in [cad7d78](https://github.com/scalar-labs/scalardl/pull/28/commits/cad7d78d87631839404ada8477a642da013affd6).